media-video/projectx/projectx-0.91.0.10-r5.ebuild

```diff
@@ -0,0 +1,127 @@
+# Copyright 1999-2024 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+JAVA_PKG_IUSE="doc source"
+
+inherit desktop java-pkg-2 java-pkg-simple toolchain-funcs xdg
+
+XDG_P="xdg-20100731"
+
+DESCRIPTION="Converts, splits and demuxes DVB and other MPEG recordings"
+HOMEPAGE="https://project-x.sourceforge.net/"
+SRC_URI="https://dev.gentoo.org/~billie/distfiles/${PF}.tar.xz
+	https://dev.gentoo.org/~billie/distfiles/${PN}-idctfast.tar.xz
+	https://dev.gentoo.org/~billie/distfiles/${XDG_P}.java.xz
+	https://dev.gentoo.org/~billie/distfiles/${PN}-icon.png"
+
+S="${WORKDIR}/Project-X"
+
+LICENSE="GPL-2"
+SLOT="0"
+KEYWORDS="~amd64 ~ppc64 ~x86"
+IUSE="X cpu_flags_x86_mmx cpu_flags_x86_sse"
+
+COMMON_DEPEND="
+	dev-java/commons-net:0
+	X? ( dev-java/browserlauncher2:1.0 )
+"
+RDEPEND="
+	${COMMON_DEPEND}
+	>=virtual/jre-1.8:*
+"
+DEPEND="
+	${COMMON_DEPEND}
+	>=virtual/jdk-1.8:*
+	app-arch/xz-utils
+"
+
+JAVA_ENCODING="iso-8859-1"
+JAVA_GENTOO_CLASSPATH="commons-net"
+JAVA_RESOURCE_DIRS=( build resources )
+JAVA_SRC_DIR="src"
+
+src_prepare() {
+	default
+
+	xdg_environment_reset
+
+	local X
+
+	# apply stdout corruption patch (zzam@gentoo.org)
+	eapply "${FILESDIR}/${PN}-0.91.0.10-stdout-corrupt.patch"
+
+	# apply BrowserLauncher2 patch
+	if use X; then
+		eapply "${FILESDIR}/${PN}-0.91.0.10-bl2.patch"
+		JAVA_GENTOO_CLASSPATH+=" browserlauncher2-1.0"
+	fi
+	rm -rf src/edu || die
+
+	# apply IDCTFast patch
+	eapply "${FILESDIR}/${PN}-0.91.0.10-idctfast.patch"
+
+	# apply XDG patch
+	cp -f "${WORKDIR}/${XDG_P}.java" "${S}/src/xdg.java" || die
+	eapply "${FILESDIR}/${PN}-0.91.0.10-xdg.patch"
+
+	# copy build.xml
+	cp -f "${FILESDIR}/build-0.91.0.10.xml" build.xml || die
```

Comment:
You removed this in the channeleditor ebuild so I think it makes sense to do so as well.
If build.xml is not required anymore it should also be removed from ${FILESDIR}.

Reply:
`build.xml` can be removed only after removal of `-r4`.